frontend/packages/integration-tests-cypress/tests/storage/snapshot.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { testName, checkErrors } from '../../support';
+import {
+  SnapshotDetails,
+  snapshotClassDropdown,
+  dropdownFirstItem,
+  pvcDropdown,
+} from '../../views/snapshot';
+import { listPage } from '../../views/list-page';
+import { PVC, testerDeployment, SnapshotClass } from '../../mocks/snapshot';
+import { detailsPage } from '../../views/details-page';
+import { submitButton } from '../../views/form';
+
+const snapshotName = `${PVC.metadata.name}-snapshot`;
+
+/**
+ * This suite works only with AWS platform.
+ */
+
+describe('Snapshot Tests', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+    cy.exec(`echo '${JSON.stringify(PVC)}' | oc apply -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc apply -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc apply -f -`);
+    // Let the PVC get bound
+    cy.wait(10 * 1000);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  it('Creates Snapshot', () => {
+    cy.clickNavLink(['Storage', 'Volume Snapshots']);
+    cy.location('pathname', { timeout: 60000 }).should(
```

Comment:
I see you added `timeout` to your `cy.location` calls, if this is going to be a common issue, please look at overwriting `cy.location` and always including the `timeout`.  Similar to: https://github.com/openshift/console/pull/6372/commits/5bf3d4c90df42c15895a4158ad717f5912dbb6af